python/paddle/fluid/tests/unittests/test_matmul_op_output_fused.py

```diff
@@ -0,0 +1,145 @@
+# Copyright (c) 2020 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import print_function
+
+import unittest
+import numpy as np
+from paddle.fluid.tests.unittests.op_test import OpTest, skip_check_grad_ci
+from ir.inference.inference_pass_test import InferencePassTest
+
+import paddle.fluid as fluid
+
+
+@skip_check_grad_ci(reason="Tests inference only optimization.")
+class TestMatMulOpSpecial(OpTest):
+    def generate_data(self):
+        self.x = np.random.random([1, 128, 128]).astype("float32")
+        self.y = np.random.random([1, 128, 64]).astype("float32")
+        self.out = np.matmul(self.x, self.y)
+        self.shape_out = []
+        self.axis_out = []
+
+    def setUp(self):
+        self.op_type = "matmul"
+        self._cpu_only = True
+        self.use_mkldnn = True
+        self.generate_data()
+
+        self.inputs = {'X': self.x, 'Y': self.y}
+        self.attrs = {'use_mkldnn': self.use_mkldnn, }
+        if len(self.shape_out) > 0:
+            self.attrs['reshape_Out'] = self.shape_out
+            self.attrs['axis_Out'] = self.axis_out
+
+        self.outputs = {'Out': self.out}
+
+    def test_check_output(self):
```

Comment:
Do you consider using `check_output_with_option` here too, I saw you use it in later functions

Reply:
The `TestMatMulOpSpecial` class inherits from OpTest class which does not have the `check_output_with_option` method. The latter tests i.e. `TestMKLDNNMatmulFuseOp` inherit from `InferencePassTest` there the method exists.